src/components/Integrations/integration-list-item/IntegrationListItem.vue

```diff
@@ -1,111 +1,72 @@
 <template>
   <div class="integration-list-item">
-
-    <el-row
-      type="flex"
-      align="middle"
-      class="info"
-    >
-      <el-col
-        :sm="24"
-      >
-        <div class="intergration-type" >
-          {{isPrivateStr}}
-        </div>
-      </el-col>
-      <el-col
-        :sm="8"
-      >
-        <el-tooltip
-          popper-class="bf-tooltip"
-          :content="userName"
-          placement="top-start"
-          :open-delay="300"
-        >
-          <avatar
-            class="icon condensed"
-            :user="user"
-          />
-        </el-tooltip>
-      </el-col>
-      <el-col
-        :sm="8"
-      >
-        {{ created }}
+    <el-row type="flex" align="middle" class="info">
+      <el-col :sm="24">
+        <div class="intergration-type">{{ isPrivateStr }}</div>
       </el-col>
-      <el-col
-        :sm="4"
-        v-if="enableSwitch"
-        class="activeSwitch"
-      >
+      <!--      <el-col-->
+      <!--        :sm="8"-->
+      <!--      >-->
+
+      <!--      </el-col>-->
+      <!--      <el-col-->
+      <!--        :sm="8"-->
+      <!--      >-->
+      <!--        {{ created }}-->
+      <!--      </el-col>-->
```

Comment:
@annaleeswitek can we remove as opposed to commenting this out?

Reply:
@edmore - sure